doc/install_ubuntu.txt

```diff
@@ -1,30 +1,33 @@
-:orphan:
-
 .. _install_ubuntu:
 
-Easy Installation of an Optimized Theano on Current Ubuntu
-==========================================================
 
-For NVIDIA Jetson TX1 embedded platform:
+Ubuntu Installation Instructions
+================================
 
-.. code-block:: bash
+There are two ways of getting the dependencies for Theano. You can either grab them
```

Comment:
Let's be as schematic as possible. Many users will just skip the text and look for links or lists. Let's give them both. Whenever you can use lists. I would do something like this (with links to the sections)

Installation:
- Through apt-get (system packages):
    - [bleeding edge (recommended)](internal link to the bleeding edge section)
    - [stable]()
- Through pip:
    - [bleeding edge]()
    - [stable]()